creator-node/docker-compose/development.env

```diff
@@ -7,7 +7,7 @@ awsBucket=
 awsAccessKeyId=
 awsSecretAccessKey=
 ipfsHost=docker.for.mac.localhost
-ipfsPort=6001
+ipfsPort=6011
```

Comment:
@SidSethi why change this?

Reply:
so that local setup can be run with discprov and cnode running distinct ipfs nodes (6001 and 6011)

Reply:
Oh i see, you're updating the ipfs.sh script to accept a optional arg for port. does that require a tooling change as well?

Reply:
this already accepts an optional arg for port btw

Reply:
not for gateway port tho

Reply:
the changes I made to the `ipfs.sh` script are to optionally expose a gateway port. that's unrelated to the api and swarm ports (5001 & 4001).
hopefully that answers your question?